### ISSUE TITLE
NTR: effector mediated suppression of plant PTI signalling

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#18683]
NTR: effector mediated suppression of plant PTI signalling

broad synonym  effector-mediated suppression of host pattern recognition receptor signalling 
effector-mediated suppression of host PRR signalling 
broad effector-mediated suppression of host PRR signaling

parent
GO:0140403   effector-mediated suppression of host innate immune response by symbiont
the PTI synonym will need to move down

That's the last one I think...

**Comment by ValWood** [comment]
GO:0039509    suppression by virus of host pattern recognition receptor activity
seems that it is describing something similar in a different way.

it  sounds like an activity- it should probably be a process like
suppression of host pattern recognition receptor signalling
I use terms for the activity MF like "receptor decoy activity" 

because then you are annotating to a process, and capturing the activity more precisely if you know the mechanism.

See what Patrick thinks.....

**Comment by ValWood** [comment]
NTR part is needed quite soon, also an ETI equivalent. There might be another ticket about this so I won't open onme.
Can we look at this on Friday?

**Comment by ValWood** [comment]
I can't find a ticket with the ETI equivalent, but we need to consolidate this part further. There are still lots of unnecessary parents....For example, should these really be "regulation of the host process" ? I thought not?  Otherwise they become annotated to the host processes (see here cell death parentage for example).
<img width="1035" alt="Screenshot 2020-01-29 at 16 59 52" src="https://user-images.githubusercontent.com/7359272/73379023-6939cc80-42b9-11ea-9128-c2bef1b72cba.png">

**Comment by ValWood** [comment]
The terms in my basket to look at


GO:0052158 | modulation by symbiont of host resistance gene-dependent defense response |  
-- | -- | --
GO:0080185 | effector-mediated induction of plant hypersensitive response by symbiont |  
GO:0140415 | effector-mediated modulation of host defenses by symbiont |  
GO:0140404 | effector-mediated modulation of host innate immune response by symbiont |  
GO:0034053 | modulation by symbiont of host defense-related programmed cell death |  


GO:0080185 | effector-mediated induction of plant hypersensitive response by symbiont | 
should be a descendant of 
GO:0034053 | modulation by symbiont of host defense-related programmed cell death |  

GO:0140404 | effector-mediated modulation of host innate immune response by symbiont |  
should be a descendant of
GO:0140415 | effector-mediated modulation of host defenses by symbiont |  

Maybe these  are in other tickets?

**Comment by pgaudet** [comment]
@ValWood I need a reference 
and in this case I would like a definition, I cannot easily build from the parent terms. 

Thanks, Pascale

**Comment by ValWood** [comment]
A process mediated by a molecule secreted by a symbiont that results in the suppression of a plant PTI signalling  ?

PMID:30584105
Title | Conserved fungal effector suppresses PAMP-triggered immunity by targeting plant immune kinases.

**Comment by ValWood** [comment]
The  intro also has many examples of  various bacterial effectors suppressing plant PTI signalling

**Comment by pgaudet** [comment]
Let me know if this is OK 

+id: GO:0140423
+name: effector-mediated suppression of pattern-triggered immunity signaling
+namespace: biological_process
+def: "A process mediated by a molecule secreted by a symbiont that results in the suppression of a pattern-triggered immunity PTI signaling pathway. PTI signalining pathways are found in plants." [PMID:30584105]
+comment: Term created see https://github.com/geneontology/go-ontology/issues/18683.
+synonym: "effector-mediated suppression of host pattern recognition receptor signaling" BROAD []
+synonym: "effector-mediated suppression of host PRR signaling" BROAD []
+synonym: "effector-mediated suppression of host PRR signalling" BROAD []
+synonym: "effector-mediated suppression of pattern-triggered immunity signaling" EXACT []
+synonym: "effector-mediated suppression of PTI signalling" EXACT []


Thanks, Pascale

**Comment by ValWood** [comment]
Yes, thanks.

Small typo "signalining"

@CuzickA  this is one of the terms we are waiting on, should com.e through to PHI canto in the next couple of days.
